artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/user/UserGroup.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.activemq.artemis.cli.commands.user;
+
+import org.apache.activemq.artemis.cli.commands.HelpAction;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "user", description = "file-based user management (example ./artemis user list)", subcommands = {ListUser.class, AddUser.class, RemoveUser.class, ResetUser.class})
```

Comment:
These group examples like "(example ./artemis user list)" show up in the shell help output, where it is actually _not_ what should be done. Not sure if it could cater to shell vs not-shell usage mode, but might be nice if it was possible.

Reply:
I think I can specify the description on the API and use the isShell that I implemented. will take a look.

Reply:
My options here are limited. I could create a different DataGroup for ShellDataGroup, or I can make a more generic text.

The other option would be some horrible hacking into the internal API.. which would be even worse. So I will go for the more generic text.

Reply:
I think I have solved this..

@gemmellr  but I will let you decide if this one could be resolved or not